katalog/fluentd/fluent.conf

```diff
@@ -1,69 +1,42 @@
-@include systemd.conf
+@include "#{ENV['FLUENTD_SYSTEMD_CONF'] || 'systemd'}.conf"
+@include "#{ENV['FLUENTD_PROMETHEUS_CONF'] || 'prometheus'}.conf"
 @include kubernetes.conf
-
-<system>
-  log_level error
-</system>
-
-<filter kubernetes.var.log.containers.nginx-ingress-**.log>
-  @type parser
-  format /^(?<host>[^ ]*) (?<domain>[^ ]*) \[(?<x_forwarded_for>[^\]]*)\] (?<server_port>[^ ]*) (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+[^\"])(?: +(?<path>[^\"]*?)(?: +\S*)?)?" (?<code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<agent>[^\"]*)")? (?<request_length>[^ ]*) (?<request_time>[^ ]*) (?:\[(?<proxy_upstream_name>[^\]]*)\] )?(?<upstream_addr>([^ ,]+(,\s)?)+) (?<upstream_response_length>([^ ,]+(,\s)?)+) (?<upstream_response_time>([^ ,]+(,\s)?)+) (?<upstream_status>([^ ,]+(,\s)?)+) (?<request_id>[^ ]*)$/
-  time_format %d/%b/%Y:%H:%M:%S %z
-  key_name log
-  types server_port:integer,code:integer,size:integer,request_length:integer,request_time:float,upstream_response_length:integer,upstream_response_time:float,upstream_status:integer
-  reserve_data yes
-</filter>
+@include conf.d/*.conf
```

Comment:
Can we have also the output configuration as an included file?

I was thinking about something like this:
```
@include "#{ENV['FLUENTD_SYSTEMD_CONF'] || 'systemd'}.conf"
@include "#{ENV['FLUENTD_PROMETHEUS_CONF'] || 'prometheus'}.conf"
@include kubernetes.conf
@include conf.d/*.conf
@include output.con
```
in this way if need to, for example, send the logs to Elasticsearch and some other log backend I can just replace `output.conf` and I don't need to repeat all the `@include` directives. What do you think?

Reply:
Makes sense to me! 👏🏻

Reply:
Uhmmmmmm
I think is not needed to have:
```

Reply:
👍

Reply:
Yes, I partly agree.

If I need to send the same output to two different log provider I am gonna do something like this in `conf.d/output.conf`:
```
<match **>
  @type copy
  <store>
    @type elasticsearch
    @id out_es
    @log_level info
    include_tag_key true
    host "#{ENV['FLUENT_ELASTICSEARCH_HOST']}"
    ...
  </store>
  <store>
    @type http
    endpoint http://logstash.acme.corp:50000
    ...
  </store>
</match>
```
Obviously this will work but it will make fluentd configuration less intuitive from my POV because I am leaving around a "zombie" output that won't receive anything more.

Reply:
I see your point... but our default behavior is just elasticsearch ;)
So, let's go as it is ;)